Utils/Filesize.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Utils;
+
+abstract class Filesize
+{
+    private final function __construct() {}
+
+    public static function getBytes($size)
```

Comment:
This is small implementation for common pretty size format, there is really no need for dependency....